contract-ui/hooks/useRouteConfig.tsx

```diff
@@ -106,12 +106,15 @@ export function useContractRouteConfig(
   });
   return [
     {
-      title: "Explorer",
+      title: "Overview",
       path: "/",
       element: () =>
         import("../tabs/overview/page").then(
           ({ CustomContractOverviewPage }) => (
-            <CustomContractOverviewPage contractAddress={contractAddress} />
+            <CustomContractOverviewPage
+              contractAddress={contractAddress}
+              contractType={contractTypeQuery?.data || undefined}
```

Comment:
what happens if there's no contractType?

Reply:
I think @shift4id is changing this as well